rpc/client_test.go

```diff
@@ -36,13 +36,14 @@ func init() {
 func TestClientHeartbeat(t *testing.T) {
 	addr := util.CreateTestAddr("tcp")
 
+	// Heartbeats are node-to-node requests: use server context for both server and client.
```

Comment:
Ah, yes. I was pondering how to make those clearer. I'll do it in a separate PR since we have TestServerContext all over the place.